lib/flags.ml

```diff
@@ -108,20 +108,22 @@ let we_are_parsing = ref false
 (* Current means no particular compatibility consideration.
    For correct comparisons, this constructor should remain the last one. *)
 
-type compat_version = V8_2 | V8_3 | V8_4 | V8_5 | Current
+type compat_version = V8_2 | V8_3 | V8_4 | V8_5 | V8_6 | Current
 
 let compat_version = ref Current
 
 let version_strictly_greater v = match !compat_version, v with
-| V8_2, (V8_2 | V8_3 | V8_4 | V8_5 | Current) -> false
-| V8_3, (V8_3 | V8_4 | V8_5 | Current) -> false
-| V8_4, (V8_4 | V8_5 | Current) -> false
-| V8_5, (V8_5 | Current) -> false
+| V8_2, (V8_2 | V8_3 | V8_4 | V8_5 | V8_6 | Current) -> false
```

Comment:
This isn't directly related to this PR But should be the object of another one. This sort of comparison can be written with a linear amount of code which would avoid big diffs like that every time a constructor gets added. See [`printer.ml`](https://github.com/coq/coq/blob/trunk/printing/printer.ml#L777-L788) for a complete example.

It would go along the lines of

```ocaml
match !compat_version, v with
| _ , Current -> false
|  Current , _ -> true
| _ , V8_6 -> false
| V8_6, _ -> true
…
```